demos/src/css/chartAuthoring-demo.css

```diff
@@ -1,33 +1,183 @@
-.floec-ca-pieChart-pie {
+.main {
+    width: 100%;
+    font-family: Verdana, Arial, sans-serif;
+}
+
+/* Main container */
+
+.floe-chartAuthoring-container {
+    width: 55em;
+    margin: 2em auto 2em auto;
+    border: 1px solid black;
+    padding: 1em 0 1em 0;
+    background-color: #FFFFFF;
+}
+
+/* Title and description inline edit containers */
+
+.floec-chartTitle {
```

Comment:
CSS classe names should start with ```floe-```. Only class names used by components or js code start with ```floec-```. The doc: https://wiki.fluidproject.org/display/docs/Class+Name+Conventions

The same comment applies to line 23, 35 and any other occurrences.